FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -79,35 +87,10 @@ TextField > .right-button:hover > .right-button-graphic {
 .password-field {
     -right-button-visible: true;
 }
-.password-field > .right-button {
-    -fx-padding: 0.166667em 0.083333em 0.166667em 0.166667em;
-}
```

Comment:
Why does this need to get removed?

Reply:
Its now the same as the normal textfield right button.